README.md

```diff
@@ -701,7 +701,7 @@ Suppose you're in a situation like this:
 - in the event that you do encounter an unexpected conflict, you are comfortable with investigating the WordPress resource queue and/or inspecting the browser DOM to identify and resolve the problem
 - the advantages of creating a subset are more important to you than the advantages of loading `all.css` or `all.js` from the Font Awesome CDN, or loading via Kit
 
-In that case, then you might prefer to do a [custom installation](https://fontawesome.com/how-to-use/customizing-wordpress/intro/getting-started) _instead_ of using this plugin package.
+In that case, then you might prefer to do a [custom installation](https://fontawesome.com/v6.0/docs/web/use-with/wordpress) _instead_ of using this plugin package.
```

Comment:
The link for custom/manual/advanced/non-plugin installation is intentionally deeper in the doc hierarchy than the top-level "use with WordPress" link. The original link here was intended to deep-link directly to that manual installation guide, saving the expert user from several drill-down clicks. But the updated link one goes to the top-level docs, from which they'd have to drill down.

When I go to that new link:
`https://fontawesome.com/v6.0/docs/web/use-with/wordpress`

And then click "Manual Installation (Advanced)" from its table of contents, it jumps me down to here on that same page:
`https://fontawesome.com/v6.0/docs/web/use-with/wordpress#manual-installation-advanced`

And that shows a section with another link into the actual advanced docs:
`https://fontawesome.com/v5.15/how-to-use/customizing-wordpress/intro/getting-started`

Notice the `v5.15` in that URL. I just attempted to change that to `v6.0` and it results in a 404.

So apparently, we don't yet have those manual installation docs available under a `v6.0` URL. In the meantime, I think that final URL is what this README should be pointing to.